lib/dor/services/client/object_metadata.rb

```diff
@@ -9,11 +9,12 @@ class Client
       class ObjectMetadata
         extend Deprecation
 
-        attr_reader :created_at, :updated_at
+        attr_reader :created_at, :updated_at, :etag
 
-        def initialize(created_at:, updated_at:)
+        def initialize(created_at:, updated_at:, etag: nil)
           @created_at = created_at
           @updated_at = updated_at
+          @etag = etag
         end
```

Comment:
I'm fuzzy on when/how we use this `ObjectMetadata` class.

Reply:
It's just a data object.